src/root-package-json.js

```diff
@@ -56,9 +56,9 @@ let scripts = {
       'lint:fix': 'npm run lint:fix --workspaces --if-present',
       prepare: 'npm run build',
       start: "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
-      'start:addon': `npm start --workspace ${addonName} -- --no-watch.clearScreen`,
-      'start:tests': `npm start --workspace ${testAppName}`,
-      test: `npm test --workspace ${testAppName}`,
+      'start:addon': `npm start --workspace ${addonName}`,
+      'start:test': `npm start --workspace ${testAppName}`,
```

Comment:
For `npm` and `pnpm`, the name `start:tests` had been used, but for `yarn`, `start:test`.

Can we use the latter, to better match the name of the `test` script`?

Reply:
imo, should probably all use `start:tests`, as folks probably have multiple tests.

`start:test-app` is also a good option, and preferred for me because it's essentially `ember s`

Reply:
Ok, I'll go ahead with `start:test-app`. I think this name reads more fluidly.